lib/tokenizers.js

```diff
@@ -195,13 +217,11 @@ export async function _readCurrentTokenizerRecord({explain = false} = {}) {
 
   const record = await collection.findOne(query, {projection});
   if(!record) {
-    const details = {
-      httpStatusCode: 404,
-      public: true
-    };
-    throw new BedrockError(
+    const error = new BedrockError(
       'Current tokenizer not found.',
-      'NotFoundError', details);
+      'InvalidStateError', {httpStatusCode: 503, public: true});
```

Comment:
At this level `NotFoundError` is correct here -- we know it's this specific error. At a higher level, where I made my other comment, the error cause could have been any number of things that ultimately led to an invalid state.

Reply:
Got it, changed it back to 404/NotFoundError.###